src/main/java/org/cryptomator/ui/error/ErrorController.java

```diff
@@ -100,8 +131,75 @@ public void copyDetails() {
 		});
 	}
 
-	/* Getter/Setter */
+	private void loadHttpResponse(HttpResponse<InputStream> response){
+		if (response.statusCode() == 200) {
+			Map<String,ErrorDiscussion> map = new Gson().fromJson(
+					new InputStreamReader(response.body(),StandardCharsets.UTF_8),
+					new TypeToken<Map<String,ErrorDiscussion>>(){}.getType());
+
+			if(!map.values().stream().filter(this::isPartialMatchFilter).findFirst().isEmpty()) {
```

Comment:
This can be simpliefied with the [`anyMatch()` function](https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/util/stream/Stream.html#anyMatch(java.util.function.Predicate))

But since we traversing the stream anyways, we can also directly filter and sort and afterwards check if there is a result. Would also elimnate the bug reported by sonarcloud.

Reply:
solved with anyMatch() function and Optional object in commit [resolved sonarclouds mentioned issues](https://github.com/cryptomator/cryptomator/pull/2953/commits/3120751d3ec2d93c8686e4fedb04c6c0b963d6c4)